hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/clustering/plan/strategy/TestSparkConsistentBucketClusteringPlanStrategy.java

```diff
@@ -133,7 +133,7 @@ public void testBuildMergeClusteringGroup() throws Exception {
     ConsistentBucketIdentifier identifier = new ConsistentBucketIdentifier(metadata);
 
     int mergeSize = (int) (maxFileSize * BUCKET_MERGE_THRESHOLD.defaultValue());
-    int[] fsSize = {0, maxFileSize, mergeSize / 2, mergeSize / 2, mergeSize / 2, maxFileSize, mergeSize / 4, mergeSize / 4};
+    int[] fsSize = {0, maxFileSize, mergeSize / 2, mergeSize / 2 + 10, mergeSize / 2, maxFileSize, mergeSize / 4, mergeSize / 4};
```

Comment:
Why is this changed?

Reply:
this was changed to fix flaky test. this change is the same as in master branch.